Tutorial1-1Basics/src/main/java/com/smarttoolfactory/tutorial1_1basics/chapter2_material_widgets/Tutorial2_5_8LazyListLayoutInfo.kt

```diff
@@ -55,9 +56,6 @@ private fun TutorialContent() {
 
         val state = rememberLazyListState()
 
-        val lazyRowWidthDp = LocalDensity.current.run { 1000f.toDp() }
```

Comment:
sorry about this modification. but i can not find the difference. could you please tell me about it ?

Reply:
In some examples i set rounded numbers specifically to show how scrolling or layouts changes. With rounded numbers instead of screen width such as 1080 it's easier to observe total width or heights. If it's some pixel value converted to dp it's for a purpose.

Reply:
thank you!